measurementlab.net

```diff
@@ -1,118 +1,59 @@
-; cPanel (update_time):1357142342 first:11.34.0.9 Cpanel::ZoneFile::VERSION:1.3 hostname:host.stevenmansour.com latest:11.34.0.11
 $ORIGIN measurementlab.net.
 $TTL 120
 
 @    IN    SOA    sns-pb.isc.org.  support.measurementlab.net.    (
-    2019050600 ;Serial Number
+    2019090300 ;Serial Number
     1h  ;refresh
     900 ;retry
     1w  ;expire
     1h ) ;minimum)
 
+;
+; NS records
+;
+@            IN    NS    sns-pb.isc.org.
+@            IN    NS    ns-mlab.greenhost.net.
 
-@        IN    NS    sns-pb.isc.org.
-@        IN    NS    ns-mlab.greenhost.net.
-@        IN    TXT   "v=spf1 include:_spf.google.com ip4:165.117.251.93 ~all"
-@        IN    TXT   "google-site-verification=lN4HHWmCV4gTtKslbFKZNrMNkYx4NhwsjSmmjsPr_Vg"
-@        IN    TXT   "google-site-verification=QzNQqgKdrElH34L_mzcEdeVoq1JP-WhiuMUB-Cm8oZg"
-@        IN    TXT   "google-site-verification=C901sQ7Uym2ZSuXbKSBsfncEXQ5tEzR4fAcAMoJnaX0"
-@        IN    TXT   "google-site-verification=JlPXKcX1J6YZfBv8xrQPE4LLWVwEm8BKu7Hv2OCW-Lw"
```

Comment:
These are needed for site ownership verification via DNS in Google Webmaster Tools and Analytics